subprojects/guides-test-fixtures/.travis.yml

```diff
@@ -15,4 +15,4 @@ before_cache:
 cache:
   directories:
     - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
\ No newline at end of file
+    - $HOME/.gradle/wrapper/
```

Comment:
This line was automatically changed by IntelliJ Editorconfig plugin because of `insert_final_newline = true` in `.editorconfig`.